### PR TITLE
suggestion to distinguish term and identifier

### PR DESCRIPTION
**Pull request by pmcb55**
Just a brain-dump draft suggestion of wording to try and help distinguish between the notions of 'term' and 'identifier'.

### CODE REVIEW

**Review by kay-kim — COMMENTED** [review]
Hey ... Thanks for this.  I quite like the content but prefer to mention only a handful of key points at the top but then have the complete discussion below.  Because, it kind of seems like in both yours and Nick's updates to the libraries,  you all are pushing the IRIs (which, of course, is a huge superset of our libraries)  over the usage of our libraries 😄 

Whereas I think just a brief intro to iri/vocab -> examples -> and then exposition would help
1) experienced users quickly find our library usage examples and pop out.
2) new users  won't be intimidated with all the text but can start to read the example first (make whatever connections they will), and then be presented with the explanation.

**Comment by pmcb55** [comment]
Sure - I'm totally fine with you re-working the wording, or re-organising the structure, but I do think it's super-important that we try to be clear about the distinction between a 'term' and the identifier for that 'term'. And the fact that 'terms' can represent classes of things (i.e. '**_the type_** of a thing' in plain English, like People, Companies, Products, Events, etc.) and properties of things (like name, address, height, altitude, etc.). It's those fundamental distinctions that weren't coming across to me at all originally, yet I think are crucial to at least sow the seed of very early on (without going into too much text).
So really, I just want to make 100% sure that **_you_** (as the tech writer) are **_fully_** aware of those **_fundamental_** distinctions.

**Comment by kay-kim** [comment]
Sounds good.  I just need to finish up couple of other things and then, I'll merge this in and fix the build errors and tweak the wording and such.
